Mathlib/Topology/Connected/PathConnected.lean

```diff
@@ -1192,6 +1197,13 @@ theorem pathConnected_subset_basis {U : Set X} (h : IsOpen U) (hx : x ∈ U) :
     (𝓝 x).HasBasis (fun s : Set X => s ∈ 𝓝 x ∧ IsPathConnected s ∧ s ⊆ U) id :=
   (path_connected_basis x).hasBasis_self_subset (IsOpen.mem_nhds h hx)
 
+theorem open_pathConnected_basis (x : X) :
```

Comment:
A theorem name should match the names of its elements. So, use `isOpen` instead of `open`, and `isPathConnected` instead of `pathConnected`. This makes it much easier to find the lemma later on if we follow this rule in a rather strict way.

Reply:
I picked this name because the name was already somewhat long, and similar lemmas like [locallyConnectedSpace_iff_open_connected_basis](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Connected/LocallyConnected.html#locallyConnectedSpace_iff_open_connected_basis) also use the shortened versions. I can change it in a moment though

Reply:
#19209 renames these two declarations, by the way.